### DISCUSSION TITLE
Jinja2 Start - End time of the query

### DISCUSSION THREAD

**Discussion by aravindh-murugesan** [jertel/elastalert2#1203] [Q&A]
Hello team,

I'm using jinja2 alert_text and try to get the timerange of the query. Could you please point me to the correct variable I should use for this

**Comment by aravindh-murugesan** [comment]
I want this to be part of my custom message as well. 

Between 2023-06-08 00:10 and 2023-06-09 00:10, there were less than 1 events.

**Comment by jertel** [comment]
The timerange isn't available due to the complexity of how alerts can be triggered from data queried much earlier than the current time. However, for some rule types you can access the timestamp field from the document that triggered the alert, and approximate the timerange of the query using that timestamp and a constant subtraction value. See #1183 as an example.
